### PR TITLE
TPMSAD-1610:In SAD layout add dropdown for languages, also Nepalis an…

### PR DESCRIPTION
**Pull request by poghosyandav**
…d Amharic langues to the existing list

Added dropDownDisplay to have option for showing languages as dropdown list.

### CODE REVIEW

File: grails-app/views/langSelector/_selector.gsp [hunk 1]
```diff
@@ -1,21 +1,13 @@
-<%@ page import="grails.plugins.GrailsPluginManager" %>
-<g:set var="locales" value="${(List<Locale>)locales}" />
-<g:set var="selected" value="${(Locale)selected}" />
+<%@ page import="grails.util.Holders; grails.plugins.GrailsPluginManager" %>
 <g:set var="hasAsset"
```
Comment (bdbogjoe) [inline grails-app/views/langSelector/_selector.gsp]: please remove this hasAsset, not needed any more in this file

**Review by bdbogjoe — CHANGES_REQUESTED** [review]

File: grails-app/views/langSelector/_selector.gsp (hunk 1 above)
Comment (poghosyandav) [reply grails-app/views/langSelector/_selector.gsp]: you mean remove it at all and don't use it ?, cause I just kept it in this general template to use in child display(bar/dropdown) templates.

Comment (bdbogjoe) [reply grails-app/views/langSelector/_selector.gsp]: still used in other templates, but not in this one

Comment (poghosyandav) [reply grails-app/views/langSelector/_selector.gsp]: ok, have done changes

**Pull request merged**